docs/services/app_api.md

```diff
@@ -0,0 +1,76 @@
+# AppAPI
+
+For [AppAPI](https://github.com/cloud-py-api/app_api) the [Docker Socket Proxy](https://github.com/cloud-py-api/docker-socket-proxy) (DSP) is required to work.
+
+## HTTP AppAPI DSP
```

Comment:
Can you also clarify when to sue http vs https? Just depending on if nextcloud is using one or the other?

Reply:
Locally for development it should be enough to use HTTP, HTTPS option in DSP is for remote install, locally could be used just for testing. In both cases ExApps are using HTTP to communicate to Nextcloud because of self-signed certs.